src/test/java/uk/gov/hmcts/befta/util/MapVerifierTest.java

```diff
@@ -126,6 +143,189 @@ public void shouldVerifySimpleMapOfAcceptableContentWithWildcards() {
+        actualBody.put("status", 400);
+        actualBody.put("error", "Bad Request");
+        actualBody.put("message", "Unknown sort direction: someInvalidSortDirection");
+        actualBody.put("message2", null);
+
+        actualBody.put("path", "/caseworkers/bfb6eeaa-cbcd-466d-aafa-07fe99e7462b/jurisdictions/AUTOTEST1"
+                + "/case-types/AAT/cases/pagination_metadata");
+        actualBody.put("details", null);
+        actualBody.put("details2", "2019-11-13");
+        actualBody.put("callbackErrors", null);
+        actualBody.put("callbackWarnings", null);
+
+        MapVerificationResult result = new MapVerifier("actualResponse", 0).verifyMap(expected, actual);
+        Assert.assertEquals(0, result.getAllIssues().size());
+        Assert.assertTrue(result.isVerified());
+    }
+
+    @Test
+    public void shouldVerifyContentWithWildcardAnythingIfExists() {
+        Map<String, Object> expected = new HashMap<>();
+        Map<String, Object> expectedBody = new HashMap<>();
+
+        expected.put("responseCode", ANY_INTEGER_NOT_NULLABLE.getValue());
+        expected.put("body", expectedBody);
+        expectedBody.put("exception", "uk.gov.hmcts.ccd.endpoint.exceptions.BadRequestException");
+        expectedBody.put("timestamp", ANY_TIMESTAMP_NOT_NULLABLE.getValue());
+        expectedBody.put("status", ANY_INTEGER_NOT_NULLABLE.getValue());
+        expectedBody.put("error", "Bad Request");
+        expectedBody.put("message", ANY_STRING_NOT_NULLABLE.getValue());
+        expectedBody.put("path", ANYTHING_PRESENT.getValue());
+        expectedBody.put("details", ANY_OBJECT_NULLABLE.getValue());
+        expectedBody.put("callbackErrors", ANYTHING_IF_EXISTS.getValue());
+        expectedBody.put("callbackWarnings", ANYTHING_IF_EXISTS.getValue());
+
+        Map<String, Object> actual = new ConcurrentHashMap<>();
+        Map<String, Object> actualBody = new HashMap<>();
+
+        actual.put("responseCode", 400);
+        actual.put("body", actualBody);
+        actualBody.put("exception", "uk.gov.hmcts.ccd.endpoint.exceptions.BadRequestException");
+        actualBody.put("timestamp", "2019-11-13T14:02:43.431");
+        actualBody.put("status", 400);
+        actualBody.put("error", "Bad Request");
+        actualBody.put("message", "Unknown sort direction: someInvalidSortDirection");
+        actualBody.put("path", "/caseworkers/bfb6eeaa-cbcd-466d-aafa-07fe99e7462b/jurisdictions/AUTOTEST1"
+                + "/case-types/AAT/cases/pagination_metadata");
+        actualBody.put("details", null);
+        actualBody.put("callbackErrors", null);
+        actualBody.put("callbackWarnings", "Test Warnings");
+
+        MapVerificationResult result = new MapVerifier("actualResponse", 0).verifyMap(expected, actual);
+        Assert.assertEquals(0, result.getAllIssues().size());
+        Assert.assertTrue(result.isVerified());
+    }
+
+    @Test
+    public void shouldVerifyContentWithWildcardAnythingNullable() {
```

Comment:
Could it be good to split these tests into more granular ones each testing a particular placeholder for full conditional coverage?

Reply:
OK.. will do that change